roles/collect/tasks/main.yml

```
@@ -8,17 +8,42 @@
     name: cost-mgmt-setup
   register: cost_mgmt_setup
 
+- name: Fail if cost-mgmt-setup does not exist
+  fail:
+    msg: 'The cost-mgmt-setup ConfigMap has not been configured.'
+  when: not cost_mgmt_setup.resources
+
 - name: Set cluster ID
   set_fact:
     ocp_cluster_id: "{{ cost_mgmt_setup.resources[0].data.clusterID }}"
+  when: cost_mgmt_setup.resources
+
+- name: Check for OCP clusterID
+  debug: msg='OCP clusterID is not defined'
+  when: not ocp_cluster_id
 
 - name: Set validation boolean
   set_fact:
     ocp_validate_cert: "{{ cost_mgmt_setup.resources[0].data.validate_cert }}"
+  when: cost_mgmt_setup.resources
+
+- name: Check for validation boolean
+  debug: msg='Validation is not defined'
```

Maybe something with more clarity?
```suggestion
  debug: msg='HTTPS certificate validation variable is not defined; defaulting to true'
```
But we could certainly use default *true*, in this case